# <mayhem_program::state::GlobalParams as anchor_lang::AccountSerialize>::try_serialize  (yours 1000 B at .text+0x25110; original 1832 B at .text+0x66eb0)
# distance 157 instructions; evidence: called-by <mayhem_program::instructions::admin::InitializeGlobalParams as anchor_lang::AccountsExit>::exit#1a95; called-by <mayhem_program::instructions::admin::Admin as anchor_lang::AccountsExit>::exit#6c17; called-by <mayhem_program::instructions::admin::WithdrawAdmin as anchor_lang::AccountsExit>::exit#f780; called-by <mayhem_program::instructions::admin::WithdrawQuoteAdmin as anchor_lang::AccountsExit>::exit#ec4a; called-by <mayhem_program::instructions::fee_recipients::InitializeFeeRecipient as anchor_lang::AccountsExit>::exit#3182; called-by <mayhem_program::instructions::fee_recipients::SweepFees as anchor_lang::AccountsExit>::exit#c62c; called-by <mayhem_program::instructions::fee_recipients::SweepFeesAdmin as anchor_lang::AccountsExit>::exit#9ba9; called-by <mayhem_program::instructions::pump_amm_interface::buy_pump_swap::BuyPumpSwap as anchor_lang::AccountsExit>::exit#c321; called-by <mayhem_program::instructions::pump_amm_interface::sell_pump_swap::SellPumpSwap as anchor_lang::AccountsExit>::exit#7e85; called-by <mayhem_program::instructions::pump_interface::buy::LegacyBuy as anchor_lang::AccountsExit>::exit#0881; called-by <mayhem_program::instructions::pump_interface::sell::LegacySell as anchor_lang::AccountsExit>::exit#757c; called-by <mayhem_program::instructions::pump_interface_v2::buy_v2::BuyV2 as anchor_lang::AccountsExit>::exit#1397; called-by <mayhem_program::instructions::pump_interface_v2::sell_v2::SellV2 as anchor_lang::AccountsExit>::exit#b39d; same-callees 2/2; neighbours between the same matched functions as in your build
--- original
+++ yours
@@ -4,55 +4,13 @@
 lddw r2, .rodata[79c1f857c3384c0b]
 mov64 r3, 8
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxw r1, [r7+232]
-stxw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
+jne r0, 0, +0x368
+mov64 r2, r7
+add64 r2, 96
 mov64 r1, r6
-mov64 r3, 4
+mov64 r3, 40
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxw r1, [r7+236]
-stxw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 4
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+96]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+104]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+112]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+120]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 ldxb r1, [r7+245]
 stxb [r10-16], r1
 mov64 r2, r10
@@ -60,18 +18,18 @@
 mov64 r1, r6
 mov64 r3, 1
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 mov64 r1, r6
 mov64 r2, r7
 mov64 r3, 32
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 mov64 r2, r7
 add64 r2, 32
 mov64 r1, r6
 mov64 r3, 32
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 ldxdw r1, [r7+64]
 ldxdw r2, [r7+72]
 stxdw [r10-8], r2
@@ -81,29 +39,27 @@
 mov64 r1, r6
 mov64 r3, 16
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+128]
-stxdw [r10-16], r1
+jne r0, 0, +0x368
+mov64 r2, r7
+add64 r2, 136
+mov64 r1, r6
+mov64 r3, 8
+call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
+jne r0, 0, +0x368
+mov64 r2, r7
+add64 r2, 220
+mov64 r1, r6
+mov64 r3, 25
+call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
+jne r0, 0, +0x368
+ldxw r1, [r7+216]
+stxw [r10-16], r1
 mov64 r2, r10
 add64 r2, -16
 mov64 r1, r6
-mov64 r3, 8
+mov64 r3, 4
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-mov64 r2, r7
-add64 r2, 244
-mov64 r1, r6
-mov64 r3, 1
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+136]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 ldxdw r1, [r7+144]
 stxdw [r10-16], r1
 mov64 r2, r10
@@ -111,7 +67,7 @@
 mov64 r1, r6
 mov64 r3, 8
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 ldxdw r1, [r7+152]
 stxdw [r10-16], r1
 mov64 r2, r10
@@ -119,55 +75,13 @@
 mov64 r1, r6
 mov64 r3, 8
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxw r1, [r7+240]
-stxw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
+jne r0, 0, +0x368
+mov64 r2, r7
+add64 r2, 160
 mov64 r1, r6
-mov64 r3, 4
+mov64 r3, 24
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+160]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+168]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+176]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+184]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+192]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 ldxdw r1, [r7+80]
 ldxdw r2, [r7+88]
 stxdw [r10-8], r2
@@ -177,15 +91,13 @@
 mov64 r1, r6
 mov64 r3, 16
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+200]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
+jne r0, 0, +0x368
+mov64 r2, r7
+add64 r2, 184
 mov64 r1, r6
-mov64 r3, 8
+mov64 r3, 24
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
+jne r0, 0, +0x368
 ldxdw r1, [r7+208]
 stxdw [r10-16], r1
 mov64 r2, r10
@@ -193,36 +105,20 @@
 mov64 r1, r6
 mov64 r3, 8
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+216]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x6a8
-ldxdw r1, [r7+224]
-stxdw [r10-16], r1
-mov64 r2, r10
-add64 r2, -16
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jeq r0, 0, +0x718
+jeq r0, 0, +0x3d8
 mov64 r1, r0
 and64 r1, 3
 mov64 r2, r1
 add64 r2, -2
-jlt r2, 2, +0x700
-jeq r1, 0, +0x700
+jlt r2, 2, +0x3c0
+jeq r1, 0, +0x3c0
 ldxdw r1, [r0+7]
 ldxdw r2, [r1]
-jeq r2, 0, +0x700
+jeq r2, 0, +0x3c0
 ldxdw r1, [r0-1]
 callx r2
 mov64 r1, 3004
 call <anchor_lang::error::Error as core::convert::From<anchor_lang::error::ErrorCode>>::from#ae9e
 exit
 mov64 r0, 2
-ja +0x710
+ja +0x3d0
